streaming/index.js

```diff
@@ -614,11 +688,14 @@ const startServer = async () => {
 
       if (!json) return;
 
-      const { event, payload, queued_at } = json;
+      const { event, payload } = json;
 
       const transmit = () => {
-        const now = new Date().getTime();
-        const delta = now - queued_at;
+        // FIXME: most stream events don't actually carry a queued_at timestamp
+        // const now = new Date().getTime();
+        // const delta = now - queued_at;
+        // streamingLatency.observe(delta);
```

Comment:
This code didn't actually work as expected, as it was usually doing `delta = now - undefined` because most events didn't carry the `queued_at` timestamp.